data/credits/en.credits

```diff
@@ -1,6 +1,48 @@
 # Lines longer than 78 characters will be wrapped automatically
 # The following line is for reference
 ##############################################################################
+(<color_cyan>0.H</color>):</color>
+<color_white>Original author:</color>    <color_white>Project Manager:</color>
+<color_yellow>Whales</color> (<color_dark_gray>retired</color>)    <color_yellow>KevinGranade</color>
+
+<color_white>Current Main Developers:</color>
+<color_yellow>I-am-Erk</color> - Ultica, lore writing, ideas guy.
+<color_yellow>KorGgenT</color> - Professional yak shaver.
+<color_yellow>BrettDong</color> - For translations and analyzing weird bugs.
+<color_yellow>jbytheway</color> - For important infrastructure work.
```

Comment:
I'm not trying to be disrespectful or belittle significance of their previous works, but I think these four esteemed contributors hardly can be considered "Current main developers" (CMD).

@I-am-Erk was active November, 2023 through January, 2024 and March, 2024 through May, 2024, but he's inactive now, and his overall amount of commits in the period of "release of 0.G - current date" is not very high.
Aside from one contribution in February, @KorGgenT hasn't any merged PRs in the whole 2024. His latest merged PR was shortly after release of 0.G in March, 2023.
@BrettDong was rather active November, 2023 through February, 2024, but he's inactive now, and his overall amount of commits in the period of "release of 0.G - current date" is not very high.
@jbytheway was rather active March, 2023 through July, 2023, but he's inactive now, and his overall amount of commits in the period of "release of 0.G - current date" is not very high.

I think we can use this github-provided graph https://github.com/CleverRaven/Cataclysm-DDA/graphs/contributors?from=03.10.2023 (don't look at the date in the link, I don't know why it works so weird). There are many contributors whose amount of commits is much higher.

Personally I propose **github-actions[bot]** to be one of our CMDs with almost 1000 commits in the period! Hurray!

Seriously though, @Maleclypse is a number one candidate for be in the "CMD" list with literally tons of merged PRs and constant work on merging other contributors' PRs.
@Standing-Storm, @GuardianDll, @RenechCDDA, @PatrikLundell, @Karol1223 has been contributing almost every day in the period and thus are also very good candidates to be the CMDs.

Reply:
FTR you should be looking at March 1 2023-Jan 1 2024 merged contributions + backports for developers active for 0.H period.

Commit count is also not a very robust indicator of amount of work done since we squash or don't with some randomness.

Reply:
> FTR you should be looking at March 1 2023-Jan 1 2024

Why? The issue is about _current_ main developers, not "developers that were active 10 months ago". What's FTR by the way?

> Commit count is also not a very robust indicator of amount of work done since we squash or don't with some randomness.

I presented commit count just as "one of" indicator, not a sole one. Before I even started typing my suggestions I also checked amount of PRs merged and what time contributors in question were active in general.

Reply:
`FTR - for the record`

Reply:
> Why? The issue is about _current_ main developers, not "developers that were active 10 months ago". What's FTR by the way?

Because they're credits for this release, otherwise we could just keep updating this constantly for experimental and keep pulling from there for new releases.

Reply:
Hey Night, we’ve changed how the 0.H branch was done to try and avoid the contributor loss that occured during previous freezes. Because the six month no content only fixes would cause lots of people to wander off and often they wouldn’t come back. So I will take a screenshot of what timeline I was pulling PRs on because it starts on the release day or 0.F and ends around the split date of the 0.H candidate branch. You’re right that doesn’t consider backports but I had a separate search looking for backports of fixes from experimental to the 0.H candidate branch and that’s where I added Procynae’s name. I don’t believe I was a merger yet at that point and this will be the first stable with your cold and flu rework if I got my timeline correct. It’s definitely a big change in method but it’s definitely helped with release burnout in the contributors.

Reply:
I still don't completely get it. Let's say we release 0.H on 1st of November. Then we should check for contributions from 0.G (1st of March, 2023) up to 1st of November, yes? No?

Reply:
We developed 0.H in the master branch from 1st of march 2023 up to 4th of january 2024; Then 0.H was moved to it's own 0.H branch, and master was considered to be 0.I; That's why 0.H would not include mentions of, for example, Katemonster, despite their giant work of adding imgui - imgui simply do not exist in 0.H
So we should cound only PRs in the main branch from 1st of march 2023 to 4th of january 2024, and then all merged pull requests that target 0.H branch
For 0.I we would need to count only PRs from 4th of january 2024, that target master branch, and then whatever gonna be backported when it would get it's own branch

Reply:
Well, that... complicates things. I wasn't aware of all that. Then you might discard my initial post here.